ui-tests-protractor/e2e/integrations/twitter-to-salesforce-integration.feature

```diff
@@ -51,6 +53,7 @@ Feature: Create integration with datamapper step
     # select salesforce connection
     When Camilla selects the "QE Salesforce" connection
     And she selects "Create or update record" integration action
+    And she selects "Contact" from "sObjectNameundefined" dropdown
```

Comment:
I've a feeling that this dropdown has a proper ID now. Can you pls double check it?
`<select _ngcontent-c25="" class="form-control ng-untouched ng-pristine ng-valid" name="sObjectName" required="" id="sObjectName">`

Reply:
Checked. You are right.